{"totalCount":8,"grade":"D","repositories":[{"url":"https://github.com/MTES-MCT/apilos","vulnerabilityAlerts":{"totalCount":8,"nodes":[{"dismissedAt":null,"createdAt":"2024-03-18T20:56:56Z","securityVulnerability":{"severity":"LOW","package":{"name":"djangorestframework-simplejwt"},"advisory":{"identifiers":[{"type":"GHSA","value":"GHSA-5vcc-86wm-547q"},{"type":"CVE","value":"CVE-2024-22513"}],"references":[{"url":"https://nvd.nist.gov/vuln/detail/CVE-2024-22513"},{"url":"https://github.com/dmdhrumilmistry/CVEs/tree/main/CVE-2024-22513"},{"url":"https://github.com/jazzband/djangorestframework-simplejwt/blob/c791e987332ed5e22a86428160d6372b1d85ffae/rest_framework_simplejwt/tokens.py#L281"},{"url":"https://github.com/advisories/GHSA-5vcc-86wm-547q"}]}}},{"dismissedAt":null,"createdAt":"2024-06-17T22:09:28Z","securityVulnerability":{"severity":"MODERATE","package":{"name":"urllib3"},"advisory":{"identifiers":[{"type":"GHSA","value":"GHSA-34jh-p97f-mpxf"},{"type":"CVE","value":"CVE-2024-37891"}],"references":[{"url":"https://github.com/urllib3/urllib3/security/advisories/GHSA-34jh-p97f-mpxf"},{"url":"https://nvd.nist.gov/vuln/detail/CVE-2024-37891"},{"url":"https://github.com/urllib3/urllib3/commit/40b6d1605814dd1db0a46e202d6e56f2e4c9a468"},{"url":"https://github.com/urllib3/urllib3/commit/accff72ecc2f6cf5a76d9570198a93ac7c90270e"},{"url":"https://github.com/advisories/GHSA-34jh-p97f-mpxf"}]}}},{"dismissedAt":null,"createdAt":"2024-06-17T22:09:50Z","securityVulnerability":{"severity":"MODERATE","package":{"name":"urllib3"},"advisory":{"identifiers":[{"type":"GHSA","value":"GHSA-34jh-p97f-mpxf"},{"type":"CVE","value":"CVE-2024-37891"}],"references":[{"url":"https://github.com/urllib3/urllib3/security/advisories/GHSA-34jh-p97f-mpxf"},{"url":"https://nvd.nist.gov/vuln/detail/CVE-2024-37891"},{"url":"https://github.com/urllib3/urllib3/commit/40b6d1605814dd1db0a46e202d6e56f2e4c9a468"},{"url":"https://github.com/urllib3/urllib3/commit/accff72ecc2f6cf5a76d9570198a93ac7c90270e"},{"url":"https://github.com/advisories/GHSA-34jh-p97f-mpxf"}]}}},{"dismissedAt":null,"createdAt":"2024-07-06T00:22:41Z","securityVulnerability":{"severity":"LOW","package":{"name":"certifi"},"advisory":{"identifiers":[{"type":"GHSA","value":"GHSA-248v-346w-9cwc"},{"type":"CVE","value":"CVE-2024-39689"}],"references":[{"url":"https://github.com/certifi/python-certifi/security/advisories/GHSA-248v-346w-9cwc"},{"url":"https://github.com/certifi/python-certifi/commit/bd8153872e9c6fc98f4023df9c2deaffea2fa463"},{"url":"https://nvd.nist.gov/vuln/detail/CVE-2024-39689"},{"url":"https://groups.google.com/a/mozilla.org/g/dev-security-policy/c/XpknYMPO8dI"},{"url":"https://security.netapp.com/advisory/ntap-20241206-0001"},{"url":"https://github.com/advisories/GHSA-248v-346w-9cwc"}]}}},{"dismissedAt":null,"createdAt":"2024-07-06T00:23:11Z","securityVulnerability":{"severity":"LOW","package":{"name":"certifi"},"advisory":{"identifiers":[{"type":"GHSA","value":"GHSA-248v-346w-9cwc"},{"type":"CVE","value":"CVE-2024-39689"}],"references":[{"url":"https://github.com/certifi/python-certifi/security/advisories/GHSA-248v-346w-9cwc"},{"url":"https://github.com/certifi/python-certifi/commit/bd8153872e9c6fc98f4023df9c2deaffea2fa463"},{"url":"https://nvd.nist.gov/vuln/detail/CVE-2024-39689"},{"url":"https://groups.google.com/a/mozilla.org/g/dev-security-policy/c/XpknYMPO8dI"},{"url":"https://security.netapp.com/advisory/ntap-20241206-0001"},{"url":"https://github.com/advisories/GHSA-248v-346w-9cwc"}]}}},{"dismissedAt":null,"createdAt":"2025-01-13T17:36:42Z","securityVulnerability":{"severity":"HIGH","package":{"name":"virtualenv"},"advisory":{"identifiers":[{"type":"GHSA","value":"GHSA-rqc4-2hc7-8c8v"},{"type":"CVE","value":"CVE-2024-53899"}],"references":[{"url":"https://nvd.nist.gov/vuln/detail/CVE-2024-53899"},{"url":"https://github.com/pypa/virtualenv/issues/2768"},{"url":"https://github.com/pypa/virtualenv/pull/2771"},{"url":"https://github.com/pypa/virtualenv/releases/tag/20.26.6"},{"url":"https://github.com/pypa/advisory-database/tree/main/vulns/virtualenv/PYSEC-2024-187.yaml"},{"url":"https://github.com/advisories/GHSA-rqc4-2hc7-8c8v"}]}}},{"dismissedAt":null,"createdAt":"2025-01-14T22:21:30Z","securityVulnerability":{"severity":"MODERATE","package":{"name":"Django"},"advisory":{"identifiers":[{"type":"GHSA","value":"GHSA-qcgg-j2x8-h9g8"},{"type":"CVE","value":"CVE-2024-56374"}],"references":[{"url":"https://nvd.nist.gov/vuln/detail/CVE-2024-56374"},{"url":"https://docs.djangoproject.com/en/dev/releases/security"},{"url":"https://groups.google.com/g/django-announce"},{"url":"https://www.djangoproject.com/weblog/2025/jan/14/security-releases"},{"url":"http://www.openwall.com/lists/oss-security/2025/01/14/2"},{"url":"https://github.com/django/django/commit/ca2be7724e1244a4cb723de40a070f873c6e94bf"},{"url":"https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2025-1.yaml"},{"url":"https://lists.debian.org/debian-lts-announce/2025/01/msg00024.html"},{"url":"https://github.com/advisories/GHSA-qcgg-j2x8-h9g8"}]}}},{"dismissedAt":null,"createdAt":"2025-01-14T22:21:30Z","securityVulnerability":{"severity":"MODERATE","package":{"name":"Django"},"advisory":{"identifiers":[{"type":"GHSA","value":"GHSA-qcgg-j2x8-h9g8"},{"type":"CVE","value":"CVE-2024-56374"}],"references":[{"url":"https://nvd.nist.gov/vuln/detail/CVE-2024-56374"},{"url":"https://docs.djangoproject.com/en/dev/releases/security"},{"url":"https://groups.google.com/g/django-announce"},{"url":"https://www.djangoproject.com/weblog/2025/jan/14/security-releases"},{"url":"http://www.openwall.com/lists/oss-security/2025/01/14/2"},{"url":"https://github.com/django/django/commit/ca2be7724e1244a4cb723de40a070f873c6e94bf"},{"url":"https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2025-1.yaml"},{"url":"https://lists.debian.org/debian-lts-announce/2025/01/msg00024.html"},{"url":"https://github.com/advisories/GHSA-qcgg-j2x8-h9g8"}]}}}]},"grade":"D"}]}